src/pages/index.ts

```diff
@@ -1,3 +1,4 @@
 export { Home } from "../pages/home/index"
 export { SearchEcoHub } from "../pages/searchHub/index"
-export { ExploreMap } from "../pages/exploreMap/index"
\ No newline at end of file
+export { ExploreMap } from "../pages/exploreMap/index"
+export { LandingPage } from "../pages/LandingPage/index"
```

Comment:
primeira letra da pasta sempre em minusculo 

```suggestion
export { LandingPage } from "../pages/landingPage/index"
```